app/src/main/java/com/checkin/app/checkin/home/epoxy/CityLocationModelHolder.kt

```diff
@@ -0,0 +1,44 @@
+package com.checkin.app.checkin.home.epoxy
+
+import android.widget.TextView
+import butterknife.BindView
+import com.airbnb.epoxy.EpoxyAttribute
+import com.airbnb.epoxy.EpoxyModelClass
+import com.airbnb.epoxy.EpoxyModelWithHolder
+import com.checkin.app.checkin.R
+import com.checkin.app.checkin.home.model.CityLocationModel
+import com.checkin.app.checkin.menu.listeners.LocationSelectedListener
+import com.checkin.app.checkin.misc.epoxy.BaseEpoxyHolder
+
+@EpoxyModelClass(layout = R.layout.item_user_location_city)
+abstract class CityLocationModelHolder : EpoxyModelWithHolder<CityLocationModelHolder.Holder>() {
+
+    @EpoxyAttribute
+    internal lateinit var data: CityLocationModel
+
+    @EpoxyAttribute
+    internal lateinit var locationSelected: LocationSelectedListener
+
+    override fun bind(holder: Holder) {
+        holder.bindData(data)
+        holder.itemView.setOnClickListener {
```

Comment:
You sure you checked other existing holders in the app? Never ever touch UI elements outside the Holder class. If you want to set click listener, go do that by overriding the 'bindView' method in the Holder classes.

Reply:
bind purpose is just to bind data. Not deal with UI elements.